### DISCUSSION TITLE
unable to fill student application

### DISCUSSION THREAD

**Discussion by CheerfulBoss** [community/community#134634] [GitHub Education]
### Hi everyone,

Hi everyone!

I'm trying to apply to GitHub Education, but the application form is proving to be a silly goose during summer break. It's August, so:

A. I'm obviously not on campus (and won't be for another almost 4 weeks) and I doubt a snap of the beach would do (I could upload a screenshot of the school calendar - no snaps though), and
B. I don't have my student card yet, nor can I show up at school to pick up a printed certificate of enrolment, since I'm in a different country - I could download one from my student page and upload it, but again - no snap options...

I don't know what to do, besides the obvious "just wait September". Can anybody help?
Thanks :)

**Comment by ens1010** [comment]
Response#1
